docs/how-to-upload-manifest.md

```diff
@@ -110,9 +75,14 @@ Each release of an operator manifest is considered immutable. If you try to push
 }
 ```
 
-If you change your operator manifest then bump up the version in `RELEASE` before pushing the manifest.
+If you change your operator artifact, you should always bump up the version of the version/release before pushing the artifact.
+
+### Delete Operator Artifacts
 
-### Delete Operator Manifest
 *You will need `admin` privileges to delete repositories.*
 
-For now, we have been deleting manifest(s) manually by going to `quay.io`. Go to the `Settings` tab of your desired repository and click the `Delete Application` button. 
\ No newline at end of file
+Operator artifacts can be deleted manually by going to `quay.io`. Go to the `Settings` tab of your desired repository and click the `Delete Application` button. This will delete all uploaded versions of your operator artifact.
+
+### Creating New Artifacts
```

Comment:
What does the paragraph below have to do with creating new artifacts? Please fix the heading.

Reply:
How about `Pushing an Operator Artifact to a New Repository`?